.github/workflows/release-prod.yaml

```diff
@@ -572,6 +565,20 @@ jobs:
         include:
           - test: TestAirgapUpgradeFromEC18
             runner: embedded-cluster-2
+          - test: TestVersion
```

Comment:
are you missing `is-lxd: true` for TestAirgapUpgradeFromEC18?

Reply:
Yeah, but looks like it's green...

Reply:
I can update in a separate pr

Reply:
nevermind. this will break on tag. will update.